tests/test_docs/helper.py

```diff
@@ -50,36 +46,23 @@ class CodeType(Enum):
     NOCODE = "nocode"
 
 
-def block_code_filter(b: Dict) -> bool:
-    """Check Mistune block is a code block."""
-    return b["type"] == MISTUNE_BLOCK_CODE_ID
-
-
-def type_filter(type_: Optional[str], b: Dict) -> bool:
-    """
-    Check Mistune code block is of a certain type.
-
-    If the field "info" is None, return False.
-    If type_ is None, this function always return true.
+def dedent_code(code: str, n_spaces: int) -> str:
+    """Removes n spaces at the beggining of each line"""
+    lines = [line[n_spaces:] for line in code.split("\n")]
+    return "\n".join(lines)
```

Comment:
To be more precise, the code removes the firs 'n' characters of each line (not necessarily spaces)